src/main/java/julia/git/Commit.java

```diff
@@ -22,6 +22,22 @@
         this.date = date;
     }
 
+    public void setParent(String parent) {
```

Comment:
These should be immutable objects. All the info should be passed in the constructor and can never be changed.